airbyte-integrations/connectors/source-newsdata/source_newsdata/spec.yaml

```
@@ -0,0 +1,192 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/newsdata
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Newsdata Spec
+  type: object
+  required:
+    - api_key
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      description: API Key
+      airbyte_secret: true
+      order: 0
+    query:
+      type: string
+      description: >-
+        Keywords or phrases to search for in the news title and content.
+        Advanced Search options:
+          - Search `Social`: query = "social"
+          - Search `Social Pizza`: query = "social pizza"
+          - Search `Social` but not with `pizza`: query = "social -pizza"
+          - Search `Social` but not with `pizza` and `wildfire`: query = "social -pizza -wildfire"
+          - Search `Social` and `pizza`: query = "social AND pizza"
+          - Search `Social` and `pizza` and `pasta`: query = "social AND pizza AND pasta"
+          - Search `Social` or `pizza`: query = "social OR pizza"
+          - Search `Social` or `pizza` but not `pasta`: query = "social OR pizza -pasta"
+          - Search `Social` or `pizza` or `pasta`: query = "social OR pizza OR pasta"
+        Note: You can't use AND and OR in the same query.
+      order: 1
+    queryInTitle:
```

```suggestion
    query_in_title:
```
Use lowercase for parameters.